### PR TITLE
Reformat code and add expected dependencies

### PR DESCRIPTION
**Pull request by cfsnsalazar**
### Description

Add Pininterest `ktlint` dependencies to project and apply formatting rules. 

- Relevant Issues : 
   * Code format

- What Changed and Why? :
   Added a `ktlint.gradle` file to encapsulate tasks necessary to reformat the code. Files can be formatted by running the command `./gradlew ktlintFormat`. 

Screen Shot 2019-09-30 at 10 08 28 PM

- Type of change :
  - [*] New feature

### CODE REVIEW

**Comment by katielevy1** [comment]
@cfsnsalazar this looks great! I'm going to take a closer look tomorrow and get back to you.

**Comment by katielevy1** [comment]
@cfsnsalazar my only ask is that you add the commands to the readme under the getting started section. Otherwise, it looks great!

**Comment by cfsnsalazar** [comment]
Awesome !!. I will write a brief summary of the gradle and ktlint commands that you can use to format your code locally.

**Comment by cfsnsalazar** [comment]
@katielevy1 just finished adding commands to the readme file. I put what I believe someone would need for a basic use.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,32 @@ You can play with the example by
...
+ - `ktlint -F`
+    <p> Use this command to re-format code that is not compliant with this project's formatting rules.
+    (if some errors cannot be fixed automatically they will be printed to stderr) </p> 
+ - `ktlint -F "src/**/*.kt"`
+    <p> Use this command to re-format certain locations (prepend ! to negate the pattern)  . 
+    (if some errors cannot be fixed automatically they will be printed to stderr) </p>    
+ - `ktlint --install-git-pre-commit-hook`
+    <p> Install git hook to automatically check files for style violations on commit. </p>    
+
+#### Use Gradle tasks ([ktlint docs](https://ktlint.github.io/)):
```
Comment (katielevy1) [inline README.md]: I would put these before the ktlint engine section and mark this as the recommended mechanism as it is much easier for people to understand.

**Comment by katielevy1** [comment]
@all-contributors add @cfsnsalazar  for code

**Pull request merged**